### ISSUE TITLE
Protected View or Read Only Mode

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28533]
### Your idea

A Mode where the score can be navigated without making any changes.  I am told there is a publish mode that may work something like this but it is not accessible.  All screen reader output, Braille output, Playback and playback controls and keyboard navigation would need to work still.

### Problem to be solved

If a blind user uses MuseScore as their main way of reading scores that have been created for them then they are often using it as a viewer to hear the score being read and played.  Currently it is very easy to get muddled between arrows up/down and alt arrows etc and accidentally change something.  Many other accidental keyboard presses could change something also and as many things do not report when they make changes the chance of accidental changes without realising is increased.  This sort of view mode would be great to review a score that is completed and you have no intention of editing in that session.  Ideally something that can be turned on and off rather than having to save it in a read only format.

### Checklist

- [x] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [x] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests
